### ISSUE TITLE
[Bug]: Screen reader is not announcing any information about graph having no data after activating filter

### ISSUE THREAD

**Issue by jamesdevshop** [microsoft/fluentui#23603]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
OS: Windows 11 version Dev (OS Build 25071.1000)
Edge Anaheim Dev: Version 101.0.1198.0 (Official build) dev (64-bit)
```

### Reproduction

https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/linechart

## Actual Behavior
When changing the filter options and graph has no content to show, then no such update is provided by screen reader.

## Expected Behavior
After activating any filter, graph of which has no data to show, screen reader should announce the related information.
For e.g: After activating filter of Past 30 days in Product Feedback tab, NVDA should announce "In-product feedback volume graph, graph has no data to display".

### Requested priority

High

### Products/sites affected

M365 admin center

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by gouttierre** [comment]
@jamesdevshop - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows.

@Raghurk - Would you be able to look at this request for charting?

**Comment by AtishayMsft** [comment]
This issue has been fixed with #26344 .
